composer.json

```diff
@@ -9,13 +9,12 @@
         "ext-xdebug": "*",
         "phpunit/phpunit": "^5.4"
     },
-    "license": "AGPL",
+    "license": "AGPL-1.0",
     "authors": [
         {
             "name": "Lewis Cowles",
             "email": "lewiscowles@me.com"
         }
     ],
-    "minimum-stability": "stable",
-    "require": {}
+    "minimum-stability": "stable"
```

Comment:
I'm assuming this is where the requires was removed, I'd like that left in (it's valid composer & json).

Reply:
Well, it's not. If you run `composer validate composer.json`, you get "Key require is a duplicate"